resources/unsupported-block-editor/external-style-overrides.css

```diff
@@ -0,0 +1,4 @@
+/* Remove \'help\' button from the editor. */
```

Comment:
Nice! Looks like the right selector in my testing. 👍

Reply:
Thank you!